### PR TITLE
added http header forwarding

### PR DESCRIPTION
**Pull request by puja108**
RFR @luebken @hectorj2f @webwurst 

I'm still not sure what else I can document for the user here, so if there's something else please tell me.

### CODE REVIEW

**Comment by marians** [comment]
My question from reading this FAQ entry: Does this affect all headers sent with the incoming request?

File: content/faq/index.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ Use the "env" key in your [service definition](/reference/swarm-json/#env).
 
 Yes. However, we currently are bound to manually setting up your certificate in our load balancer. Just ping us and we'll get on it. It only takes a few minutes.
 
+## Can I enable HTTP header forwarding for requests to my services?
+
+Yes. This feature works on a per cluster basis and is enabled on some of our clusters already. If your cluster doesn't have header forwarding enabled, yet, just ping us and we can enable it for you.
```
Comment (JosephSalisbury) [inline content/faq/index.md]: I'd remove the comma after enabled, so it's "doesn't have header forwarding enabled yet, just ping us"

**Comment by luebken** [comment]
I would be more specific e.g. http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/x-forwarded-headers.html

**Comment by puja108** [comment]
changed it a bit to be more specific about it. Not sure if that's enough. I don't want to include a whole section on what the stuff actually is in a FAQ.

**Comment by hectorj2f** [comment]
@marians Of course, if we add HTTP headers in the shared cluster they will be added to all incoming requests. We could use somehow rules to filter stuff but that won't scale at all. 
On the other hand, if a customer uses a private cluster that won't happen as they will have their own LBs

**Comment by marians** [comment]
@puja108 Seems like this was fine content-wise, but we never merged it. Do you still want to? If yes, I suggest also changing the last modified date in the frontmatter.

**Comment by marians** [comment]
Thanks @puja108!

**Pull request merged**